recipe/recipe.yaml

```
@@ -46,7 +46,9 @@ source:
 
 build:
   number: ${{ build_number }}
-  skip: win
+  skip:
+    - win
+    - osx and x86_64
```

It would be good to document why
```suggestion
     # newer macOS needs root >=6.34.8, see
     # https://github.com/root-project/root/issues/18429
    - osx and x86_64
```